liquibase-core/src/test/groovy/liquibase/database/core/SybaseDatabaseTest.groovy

```diff
@@ -3,83 +3,111 @@ package liquibase.database.core
 import liquibase.CatalogAndSchema
 import liquibase.Scope
 import liquibase.database.DatabaseConnection
+import liquibase.exception.DatabaseException
 import liquibase.executor.Executor
 import liquibase.executor.ExecutorService
 import liquibase.statement.core.GetViewDefinitionStatement
 import spock.lang.Specification
 import spock.lang.Unroll
 
-public class SybaseDatabaseTest  extends Specification {
+public class SybaseDatabaseTest extends Specification {
 
     def cleanup() {
-		Scope.currentScope.getSingleton(ExecutorService.class).reset()
+        Scope.currentScope.getSingleton(ExecutorService.class).reset()
     }
 
-	def testIsSybaseProductName() {
+    def testIsSybaseProductName() {
         when:
-		def SybaseDatabase database = new SybaseDatabase()
+            def SybaseDatabase database = new SybaseDatabase()
 
         then:
-		assert database.isSybaseProductName("Sybase SQL Server")
-		assert database.isSybaseProductName("sql server")
-		assert database.isSybaseProductName("ASE")
-		assert database.isSybaseProductName("Adaptive Server Enterprise")
-	}
+            assert database.isSybaseProductName("Sybase SQL Server")
+            assert database.isSybaseProductName("sql server")
+            assert database.isSybaseProductName("ASE")
+            assert database.isSybaseProductName("Adaptive Server Enterprise")
+    }
 
     @Unroll
-	def getViewDefinition() throws Exception {
+    def getViewDefinition() throws Exception {
         when:
-        def executor = Mock(Executor)
-        executor.queryForList(_ as GetViewDefinitionStatement, String.class) >> viewRows
-		SybaseDatabase database = new SybaseDatabase()
-		Scope.currentScope.getSingleton(ExecutorService.class).setExecutor("jdbc", database, executor)
+            def executor = Mock(Executor)
+            executor.queryForList(_ as GetViewDefinitionStatement, String.class) >> viewRows
```

Comment:
This PR taught me that "_ is a special character in Spock and it means any , it is used as an argument matcher, in this case it is basically ensuring the argument  is indeed GetViewDefinitionStatement class."

Thank you, @suryaaki2 !